bitnami/spark/templates/configmap.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if and (not .Values.worker.configurationConfigMap) (not .Values.master.configurationConfigMap) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "spark.fullname" . }}-config
+  labels:
+    app.kubernetes.io/name: {{ include "spark.name" . }}
+    helm.sh/chart: {{ include "spark.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: configmap
+data:
+  spark-env.sh: |-
+    {{- if .Values.master.host }}
+    SPARK_MASTER_HOST=${SPARK_MASTER_HOST}
```

Comment:
shouldn't this be `SPARK_MASTER_HOST={{ .Values.master.host }}`

The same for other env vars